ManPhamQuang/COSC2769-rmit
# Feature: Chat Message

### Changelog
- Integrate Twilio Chat Message
- Basic UI for Chat 
- Able to post a message
- Fetch all messages from a current room

### How to test
1. Start a Backend server
2. Start FrontEnd Server
3. Go to http://localhost:3000/room/video
4. Enter your name and Name room
5. Connect

Verify that we can join a video call room.
Verify that we can send a message

7. Open a new tab and open http://localhost:3000/room/video
8. Enter your new name and the same **Name Room**
9. Connect
Verify that we can see all previous message

### Screenshot
Please note that I intentionally disable the video for testing. It should work in the PR.

<img width="1392" alt="Screen_Shot_2021-08-03_at_16_31_13" src="https://user-images.githubusercontent.com/5878421/127995737-2f388b74-62c7-4bd4-86cc-ab78ef41c991.png">
